my-app/src/store/usersCreateslice.ts

```diff
@@ -0,0 +1,31 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { IUser, InitialState } from '../core/interfaces/interfaces';
+import { fetchLeaders } from '../core/services/getRequest';
+
+const initialState: InitialState = {
+	day: 0,
+	usersLeaders: [],
+	topUsers: [],
+};
+
+const users = createSlice({
+	name: 'users',
+	initialState,
+	reducers: {},
+	extraReducers: builder => {
+		builder.addCase(fetchLeaders.fulfilled, (state, action: PayloadAction<Array<IUser>>) => {
+			// eslint-disable-next-line no-param-reassign
+			state.usersLeaders = [...state.usersLeaders, action.payload];
+			// eslint-disable-next-line no-param-reassign
```

Comment:
You can disable this rule for the entire file in case you see that you write a command for disabling a lot of times

Reply:
i added a new rules in ESlint configurator and have delete all comments